Test/TestCase/SearchManagerTest.php

```diff
@@ -153,17 +165,154 @@ public function testRemove()
         $id = $view->getId();
         $searchManager->remove($view);
         $searchManager->flush();
+        sleep(1);
```

Comment:
You can avoid the sleep by using the EntityRepository::delete() method with refresh.